docs/platforms/react-native/tracing/instrumentation/react-native-navigation.mdx

```diff
@@ -33,11 +29,26 @@ Sentry.init({
 You can configure the instrumentation by passing an options object to the constructor:
 
 ```javascript
-new Sentry.ReactNativeNavigationInstrumentation({
-  routeChangeTimeoutMs: 1000, // default: 1000
+Sentry.reactNativeNavigationIntegration({
+  navigation: Navigation, // Navigation from `react-native-navigation`
+  routeChangeTimeoutMs: 1_000, // default: 1_000
+  enableTabsInstrumentation: true, // default: false
+  ignoreEmptyBackNavigationTransactions: true, // default: true
 });
 ```
 
+### navigation
+
+This option is required and should be the `Navigation` object imported from the navigation library.
+
 ### routeChangeTimeoutMs
 
-This option specifies how long the instrumentation will wait for the route to mount after a change has been initiated before the transaction is discarded. The default value is 1000ms.
+This option specifies how long the instrumentation will wait for the route to mount after a change has been initiated before the transaction is discarded. The default value is `1_000`.
+
+### enableTabsInstrumentation
+
+Instrumentation will create a transaction on tab change. By default only navigation commands create transactions. The default value is `false`.
+
+### ignoreEmptyBackNavigationTransactions
+
+Does not sample transactions that are from routes that have been seen any more and don't have any spans. This removes a lot of the clutter as most back navigation transactions are now ignored. The default value is `true`.
```

Comment:
```suggestion
This ensures that transactions that are from routes that are no longer being seen and don't have any spans, are not being sampled. This removes a lot of clutter, making it so that most back navigation transactions are now ignored. The default value is `true`.

```

Reply:
```suggestion
This ensures that transactions that are from routes that've been seen and don't have any spans, are not being sampled. This removes a lot of clutter, making it so that most back navigation transactions are now ignored. The default value is `true`.
```

It looks good, but I noticed I had a mistake in the original.